packit_service/worker/jobs.py

```diff
@@ -615,7 +631,8 @@ def get_jobs_matching_event(self) -> List[JobConfig]:
                 )
                 and job not in jobs_matching_trigger
             ):
-                jobs_matching_trigger.append(job)
+                if not job.manual_trigger or self.event in MANUAL_EVENTS:
```

Comment:
as I am looking into the list of MANUAL_EVENTS, we could reduce the events to only `AbstractCommentEvent`, and `CheckRerunEvent` there and use `any(isinstance(self.event, event_type) for event_type in MANUAL_EVENTS)` here to count with inheritance